src/set_units.jl

```diff
@@ -309,7 +309,7 @@ function mass(species::Species, unit::Union{Unitful.FreeUnits,AbstractString}=ge
   if dimension(unit) != dimension(u"kg")
     throw(ErrorException("unit have proper dimension"))
   end
-  return (species.mass_in_eV * u"eV/c^2" |> unit).val
+  return (species.mass_in_eV |> unit).val
```

Comment:
Both `mass` and `charge` should be switched to `massof` and `chargeof`. And if `nothing` is passed, then use `upreferred`. E.g.

```julia
function massof(species::Species, unit::Union{Nothing, Unitful.FreeUnits}=nothing)
  if isnothing(unit)
    return upreferred(species.mass)
  else
    return uconvert(species.mass, unit)
  end
end
```
Likewise for `chargeof`